### ISSUE TITLE
Sort buttons/dropdowns on category pages

### ISSUE THREAD

**Issue by jshearon** [nss-evening-cohort-12/pants-overalls#5]
# USER STORY
As a user when I want to organize the item list from lowest price to highest price, I will see a button that allows me to sort the items below when I click on it.   

# AC
WHEN I select something in the dropdown list
THEN I should see the cards below sorted by price

# DEV NOTES
Two sort options should be sort low to high and high to low
The array will be sorted using the sort() method with a compare function that sorts the array by the price key value.

-- BOOTSTRAP DROPDOWN--
it is the first dropdown on the link.

![Screen Shot 2020-05-05 at 7 45 05 PM](https://user-images.githubusercontent.com/62916291/81129512-2fa4dd00-8f0a-11ea-87f3-447e94c99e88.png)
![Screen Shot 2020-05-11 at 7 28 57 PM](https://user-images.githubusercontent.com/62916291/81625186-bbf94900-93bd-11ea-9405-adf3b6abf75c.png)

**Comment by luketlancaster** [comment]
where is this dropdown going to live? Is this in the nav? Or just a button somewhere else?

fyi you can link to issues by typing `#` and the number of the issue, i.e. #1. So if this is related to another issue, do that to link the two things together. Like "this depends on issue #2 so make sure that's done first, then put this in there" or whatever

**Comment by michaelclark2** [comment]
See luke's comment above, i dont think that was addressed.  talk more about how you plan to compare each object, are you going to use a certain javascript method?
